features/bevragen/persoon/verblijfplaats/dev/onbekendwaardes-gba.feature

```diff
@@ -0,0 +1,120 @@
+# language: nl
+
+
+Functionaliteit: properties die gevuld zijn met standaard-waardes worden ongewijzigd geleverd voor de verblijfplaats. 
+
+  Wanneer in de registratie specifieke waarden gereserveerd zijn voor een onbekende waarde, worden deze waarden ongewijzigd geleverd door de GBA-API.
+  Wanneer een element in de registratie een standaardwaarde heeft, die betekent dat de waarde onbekend is, wordt het corresponderende veld opgenomen in de response met de betreffende waarde. 
+
+
+Rule: de volgende standaardwaarden worden ongewijzigd geleverd bij de verblijfplaats
+  Het gaat om de volgende properties en standaardwaardes van de persoon:
+  | property                                         | standaardwaarde  |
+  | ------------------------------------------------ | ---------------- |
+  | verblijfplaats.straat                            | .                |
+  | verblijfplaats.huisnummer                        | 0                |
+  | verblijfplaats.woonplaats                        | .                |
+  | verblijfplaats.nummeraanduidingIdentificatie     | 0000000000000000 |
+  | verblijfplaats.adresseerbaarObjectIdentificatie  | 0000000000000000 |
+
+  Scenario: onbekend waarde "." voor straat
+    Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende 'verblijfplaats' gegevens
+    | naam                  | waarde |
+    | functie adres (10.10) | W      |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam               | waarde |
+    | gemeente_code      | 0518   |
+    | straatnaam (11.10) | .      |
+    | huisnummer (11.20) | 1      |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000164                           |
+    | fields              | burgerservicenummer,verblijfplaats  |
```

Comment:
ook bij de gba scenario's zou ik alleen het veld met standaardwaarde vragen. Bijkomend voordeel hier is dat je dan ook beschrijft/valideert dat als de gba-api om een veld met standaardwaarde wordt gevraagd ook alle andere verblijfplaats velden wordt geleverd